### DISCUSSION TITLE
Edge has dropped support for manifest v2

### DISCUSSION THREAD

**Discussion by gera2ld** [violentmonkey/violentmonkey#1567] [General]
![image](https://user-images.githubusercontent.com/3139113/190864432-cae54c7f-4460-4b29-bae8-b3879e3f2533.png)

I guess we are not able to support Edge any more.

**Comment by tophf** [comment]
According to their [roadmap](https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/manifest-v3) they forbid MV2 only for new extensions, not for updates, so it might be a bug in their dashboard.

**Reply by gera2ld** [reply]
Yes, you are right. I managed to update it after reloading the page.
